### PR TITLE
Replace topaz peaks file

### PR DESCRIPTION
**Pull request by NickDraper**
Replace slow Isaw Peaks file with a nexus translation of the same peaks workspace.  Many of the algorithms that use this have been altered to use this instead.  Loading the same peaks workspace from a processed nexus file is 2x faster.

**To test:**
1. Unit tests must pass on all environments

re #19866. 

**Release Notes** 
*Does not need to be in the release notes.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####

- [ ] Is the code of an acceptable quality?
- [ ] Does the code conform to the [coding standards](http://www.mantidproject.org/Coding_Standards)?
- [ ] Are the unit tests small and test the class in isolation?
- [ ] If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- [ ] Do changes function as described? Add comments below that describe the tests performed?
- [ ] Do the changes handle unexpected situations, e.g. bad input?
- [ ] Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: Framework/Crystal/test/FindUBUsingLatticeParametersTest.h [hunk 1]
```diff
@@ -171,14 +172,14 @@ class FindUBUsingLatticeParametersTest : public CxxTest::TestSuite {
...
-    TS_ASSERT_DELTA(latt.c(), 8.6566, 5e-4);
-    TS_ASSERT_DELTA(latt.alpha(), 92.6267, 5e-4);
-    TS_ASSERT_DELTA(latt.beta(), 103.7440, 5e-4);
-    TS_ASSERT_DELTA(latt.gamma(), 90.0272, 5e-4);
+    TS_ASSERT_DELTA(latt.a(), 13.9520, 1e-2);
+    TS_ASSERT_DELTA(latt.b(), 19.5145, 1e-2);
+    TS_ASSERT_DELTA(latt.c(), 8.6566, 1e-2);
+    TS_ASSERT_DELTA(latt.alpha(), 92.6267, 1e-2);
+    TS_ASSERT_DELTA(latt.beta(), 103.7440, 1e-2);
+    TS_ASSERT_DELTA(latt.gamma(), 90.0272, 1e-2);
```
Comment (SimonHeybrock) [inline Framework/Crystal/test/FindUBUsingLatticeParametersTest.h]: What is the reason for the adapted tolerances?

**Review by SimonHeybrock — CHANGES_REQUESTED** [review]
Looks good, just one question:

File: Framework/Crystal/test/FindUBUsingLatticeParametersTest.h (hunk 1 above)
Comment (NickDraper) [inline Framework/Crystal/test/FindUBUsingLatticeParametersTest.h]: The altered file lead to slightly different values in the test, It is not completely clear why, but the algorithm is using optimisation so it is likely that a slight difference in precision of one of the starting parameters lead to a slightly different minima.  The code of the algorithm itself was not changed at all, so I'm confident we have not introduced any errors.

Comment (SimonHeybrock) [inline Framework/Crystal/test/FindUBUsingLatticeParametersTest.h]: Ok, that makes sense. I hope it does not point to a deeper underlying issue.

**Review by SimonHeybrock — APPROVED** [review]

**Pull request merged**